doc/md/cloud/agents.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Drift Detection
+id: agents
+slug: /cloud/agents
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Background
+
+Some features require Atlas Cloud to connect to your database. As it is uncommon for databases to be directly accessible
+from the internet, you can run Atlas Agents in your database's network to facilitate this communication. Agents register
+themselves via credentials against your Atlas Cloud account and continuously poll it for work.
+
+The following features require an Atlas Agent installed in your databases network:
+* Drift Detection
+* Cloud mediated deployments (coming soon)
+* Schema monitoring and auditing (coming soon)
+
+:::info PREMIUM FEATURE
+Drift Detection is currently only available in the premium subscription.
+:::
+
+### Create an Agent
+
+The first step to establish a connection between Atlas and your database, is telling Atlas about your intention to do
+so. Head over to the general settings and click on the "Agents" tab.
+
+:::info
+Only admins can create agents. Ensure you have sufficient permissions before proceeding.
+:::
+
+<details>
+<summary>Screenshot Example</summary>
+
+![](https://atlasgo.io/uploads/cloud/drift-detection/1.png)
+</details>
+
+Pick a name for your agent and hit **Create Agent**.
+
+
+<details>
+<summary>Screenshot Example</summary>
+
+![](https://atlasgo.io/uploads/cloud/drift-detection/2.png)
+</details>
+
+### Connect the Agent
+
+Now that we have created an agent in Atlas, we need to run the agent binary in our VPC and check if it can access the
+cloud. To authenticate against Atlas, a token will be created which you will need to provide the agent with. Store this token
+in a secure place, as you won't be able to see it again. However, you can always create a new one.
+
+<details>
+<summary>Screenshot Example</summary>
+
+![](https://atlasgo.io/uploads/cloud/drift-detection/3.png)
+</details>
+
+If the agent has access to the internet and can reach Atlas, you should see a success message.
```

Comment:
The image isn't enough here, add some more instructions on how to actually start the agent. In this section you will also add Kubernetes, ECS, etc..